tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```diff
@@ -409,82 +409,82 @@ em {
 }
 
 .deadsay {
-  color: #732FCD;
+  color: #8041D8;
 }
 .radio{
-  color: #008000;
+  color: #158C15;
 }
 .deptradio{
-  color: #ff00ff;
+  color: #B21AB2;
 }
 .comradio{
-  color: #395A9A;
+  color: #3E67B2;
 }
 .syndradio{
-  color: #6D3F40;
+  color: #7F4648;
 }
 .centradio{
-  color: #5C5C8A;
+  color: #636399;
 }
 .airadio{
-  color: #FF00FF;
+  color: #B21AB2;
 }
 
 .casradio{
-  color: #A30000;
+  color: #A51830;
 }
 .engradio{
-  color: #A66300;
+  color: #CC631E;
 }
 .medradio{
-  color: #008160;
+  color: #158C78;
 }
 .sciradio{
-  color: #993399;
+  color: #993599;
 }
 
 .supradio{
-  color: #5F4519;
+  color: #996F2D;
 }
 
 .alpharadio{
-  color: #EA0000;
+  color: #BF2F2F;
 }
 .bravoradio{
-  color: #C68610;
+  color: #CC8814;
 }
 .charlieradio{
-  color: #AA55AA;
+  color: #B259B2;
 }
 .deltaradio	{
-  color: #3A59D9;
+  color: #3656D8;
 }
 
 .binarysay{
-  color: #20c20e;
+  color: #24BF13;
   background-color: #000000;
   display: block;
 }
 .binarysay a{
-  color: #00ff00;
+  color: #56BF56;
 }
 .binarysay a:active, .binarysay a:visited{
-  color: #88ff88;
+  color: #56BF56;
 }
 
 .alert{
-  color: #ff0000;
+  color: #FF0000;
 }
 h1.alert, h2.alert{
-  color: #99aab5;
+  color: #99AAB5;
```

Comment:
FUCKING READ THIS : http://www.byond.com/docs/ref/#/list/associations

AND MAYBE YOU WILL STOP DOING SHIT LIKE THIS